tests/integration_tests/performance/test_versioned_serialization_benchmark.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""A test that ensures that all unit tests pass at integration time."""
+
+import platform
+import os
+import logging
+import json
+import shutil
+
+import pytest
+import framework.utils as utils
+import host_tools.proc as proc
+from framework.defs import FC_WORKSPACE_DIR
+
+BENCHMARK_DIRECTORY = "{}/src/vmm".format(FC_WORKSPACE_DIR)
+
+PROC_MODEL = proc.proc_type()
+
+BASELINES = {
```

Comment:
How were these baselines computed?

Reply:
I have computed the baselines based on the performance we receive with the current implementation.